dnf-behave-tests/dnf/steps/module.py

```diff
@@ -46,19 +47,22 @@ def check_module_list(context):
 
 
 def get_modules_state(installroot):
-    cfgdir = os.path.join(installroot, 'etc/dnf/modules.d')
-    cfg = ConfigParser()
-    cfg.read(glob.glob(cfgdir + '/*.module'))
```

Comment:
Since the `ConfigParser` and `glob` are no longer used should we also drop their imports?

Reply:
Good catch! I removed them.